content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,214 @@
+
+ - **Carrier name.** This is a mandatory field, when trying to proceed to the next step without inputting this field, the following error message is displayed: "The name field is required." When trying to enter invalid characters to this field, and trying to proceed to the next step, the following error message is displayed: "The name field is invalid".
+The following tooltip is displayed: "Allowed characters: letters, spaces and "().-". The carrier's name will be displayed during checkout. For in-store pickup, enter 0 to replace the carrier name with your shop name.".
+
+ - **Transit time.** This field is mandatory, when trying to proceed to the next step without inputting this field, the following error message is displayed: "The field delay is required at least in _default language_". This field can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+The following tooltip is displayed: "The delivery time will be displayed during checkout.".
+
+ - **Speed grade.** Only one number from 0 to 9 is allowed in this field. If an incorrect character is filled, the following error message is displayed: "The grade field is invalid.". If more than 1 number is filled, the following error message is displayed: "The grade field is too long (1 chars max).".
+The following tooltip is displayed: "Ented a "0" for a longest shipping delay, or "9" for the shortest shipping delay.".
+
+ - **Logo.** This field allows uploading a logo. To upload a logo, there is a button on the right of the field named "Choose a file". A help text is displayed under the field: "Format: JPG, GIF, PNG. Filesize: 8.00 MB max. Current size: undefined.". When trying to upload an image with a wrong format or > 8MB, the following error message is displayed: "Cannot upload file".
+
+ - **Tracking URL.** A help text is displayed under the field: "For example: 'http://example.com/track.php?num=@' with '@' where the tracking number should appear.". 
+A tooltip is displayed: "Delivery tracking URL: Type '@' where the tracking number should appear. It will be automatically replaced by the tracking number." The URL should be written as displayed in the tooltip, otherwise, the following error message is displayed: "The url field is invalid.".
+
+## Call to action
+ 
+ - **Previous.** In the first step, this button is disabled.
+ - **Next.** If there is an error, this button is disabled. After clicking on it, the next step is displayed.
+ - **Finish.** If there is an error, this button is disabled. After clicking on it, the carrier is saved and the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page is displayed with the following success message: "Successful creation" if a carrier was added or "Successful update" if a carrier was edited.
+
+# Multistore (2nd step)
```

Comment:
Both call to action logo and steps are described at the begining. Could you please remove them from this section?